Profiler/utils/Ipc.cpp

```diff
@@ -0,0 +1,145 @@
+#ifdef TIA
+#include "Ipc.h"
+
+#include <zmq.h>
+
+#define IPC_TIMEOUT_MS 250
```

Comment:
Where does this number come from? Maybe this is the culprit in the distributed setting, because the profiler just doesn't wait long enough for the request to return?

Reply:
Made this option configurable.